IMPLEMENTATION_COVERAGE.md

```diff
@@ -1,3 +1,139 @@
+- [acm - 41% implemented](#acm---41--implemented)
+- [acm-pca - 0% implemented](#acm-pca---0--implemented)
+- [alexaforbusiness - 0% implemented](#alexaforbusiness---0--implemented)
+- [apigateway - 24% implemented](#apigateway---24--implemented)
+- [application-autoscaling - 0% implemented](#application-autoscaling---0--implemented)
+- [appstream - 0% implemented](#appstream---0--implemented)
+- [appsync - 0% implemented](#appsync---0--implemented)
+- [athena - 0% implemented](#athena---0--implemented)
+- [autoscaling - 44% implemented](#autoscaling---44--implemented)
+- [autoscaling-plans - 0% implemented](#autoscaling-plans---0--implemented)
+- [batch - 93% implemented](#batch---93--implemented)
+- [budgets - 0% implemented](#budgets---0--implemented)
+- [ce - 0% implemented](#ce---0--implemented)
+- [cloud9 - 0% implemented](#cloud9---0--implemented)
+- [clouddirectory - 0% implemented](#clouddirectory---0--implemented)
+- [cloudformation - 56% implemented](#cloudformation---56--implemented)
+- [cloudfront - 0% implemented](#cloudfront---0--implemented)
+- [cloudhsm - 0% implemented](#cloudhsm---0--implemented)
+- [cloudhsmv2 - 0% implemented](#cloudhsmv2---0--implemented)
+- [cloudsearch - 0% implemented](#cloudsearch---0--implemented)
+- [cloudsearchdomain - 0% implemented](#cloudsearchdomain---0--implemented)
+- [cloudtrail - 0% implemented](#cloudtrail---0--implemented)
+- [cloudwatch - 56% implemented](#cloudwatch---56--implemented)
+- [codebuild - 0% implemented](#codebuild---0--implemented)
+- [codecommit - 0% implemented](#codecommit---0--implemented)
+- [codedeploy - 0% implemented](#codedeploy---0--implemented)
+- [codepipeline - 0% implemented](#codepipeline---0--implemented)
+- [codestar - 0% implemented](#codestar---0--implemented)
+- [cognito-identity - 0% implemented](#cognito-identity---0--implemented)
+- [cognito-idp - 34% implemented](#cognito-idp---34--implemented)
+- [cognito-sync - 0% implemented](#cognito-sync---0--implemented)
+- [comprehend - 0% implemented](#comprehend---0--implemented)
+- [config - 0% implemented](#config---0--implemented)
```

Comment:
Yours truly added a bunch of AWS Config support. A lot of the data here is not accurate.